packages/shared/styles/components/SfNotification.scss

```diff
@@ -0,0 +1,93 @@
+@import '../variables';
+$notification__message-padding-left: 0.625rem !default;
+$notification--info-color: $c-dark-primary !default;
```

Comment:
info, warning, alert, success colors are all defined as scss vars

Reply:
I couldn't find alert color in scss vars so I used blue color from figma design.  Warning is defined as yellow, but according to figma it should be red so I used color from figma.  Info  is blue according to scss vars so I used $c-dark-primary var. Is that okey?